collectors/crowdstrike/cfn/README.md

```diff
@@ -0,0 +1,49 @@
+# crowdstrike-collector.template
+
+Alert Logic Crowdstrike Log Collector CloudFormation template.
+
+ 
+# Before you begin
+
+This procedure requires administrative permissions in AWS and your Alert Logic 
+Cloud Insight account. You also need to download the Alert Logic custom [CFT](https://github.com/alertlogic/paws-collector/blob/master/collectors/crowdstrike/cfn/crowdstrike-collector.template).
+
+
+# Installation
+
+Currently, we support US regions only: `us-east-1`, `us-east-2`, `us-west-1`, 
+`us-west-2`.
+
+
+## CFN Template
+1. Go to AWS CloudFormation 
+1. Click on *Create Stack* and use following S3 URL with cfn template.
+`https://s3.amazonaws.com/alertlogic-collectors-us-east-1/cfn/crowdstrike-collector.template`
+1. In the next step, you will need:
+   - `Stack name` - Any name you have used for creating an AWS stack
+   - `AlApiEndpoint` - use predefined `api.global-services.global.alertlogic.com` 
+   - `AlApplicationId` - use `crowdstrike` (Alert Logic Application Id for collector logs)  
+   - `AlDataResidency` - use `default`
+   - `AlertlogicAccessKeyId` - `access_key_id` returned from AIMS
+   - `AlertlogicCustomerId` - Optional. Alert Logic customer ID which collected data should be reported for. If not set customer ID is derived from AIMs tokens
+   - `AlertlogicSecretKey` - `secret_key` returned from AIMS
+   - `CrowdstrikeAPINames` - Define API names. Please pass JSON formatted list. Possible values are ["Incident", "Detection"]
+   - `CrowdstrikeClientId` - Crowdstrike Client ID 
+   - `CrowdstrikeEndpoint` - Crowdstrike Endpoint
+   - `CrowdstrikeSecret` - Crowdstrike Secret 
+   - `CollectionStartTs` - example `2019-11-21T16:00:00Z` Timestamp when log collection starts
+   - `CollectorId` - default `none` Optional. A collector UUID if known.
+   - `PackagesBucketPrefix` - example `alertlogic-collectors` S3 bucket name prefix where collector packages are located.
+   - `PawsCollectorTypeName` - use `crowdstrike`
```

Comment:
I think we don't have this param in crowdstrike template. Please double -check

Reply:
Done